apps/storefront/lib/auth/Readme.md

```diff
@@ -68,7 +68,7 @@ const { isAuthenticating, signIn, signOut, checkoutSignOut } = useSaleorAuthCont
 
 #### **`useAuthChange({ onSignedIn, onSignedOut }: UseAuthChangeProps) => void`**
 
-Once the successful sign in or sign out happens, as well as failed authentication (for e.g when refresh token is expired and there's no way to obtain a new access token from the api) you'll probably want to trigger some actions. Hence we're provided a `useAuthChange` hook that'll listen to storage changes and trigger callbacks.
+Regardless of whether the sign-in or sign-out is successful, you probably want to react to it. Hence we provided a `useAuthChange` hook that listens to storage changes and triggers callbacks.
```

Comment:
I don't think the `Regardless of whether sign in or sign out is successful` is accurate here considering that the hook won't run in case of an unsuccessful sign-out. It'll run if you:
- signed in successfully (onSignedIn)
- signed out successfully or your tokens expired and there is no way to refresh them (onSignedOut)